src/js/routes/services.js

```diff
@@ -1,37 +1,69 @@
 import Router from 'react-router';
-let Route = Router.Route;
+let {Route, Redirect} = Router;
 
+import DeploymentsTab from '../pages/services/DeploymentsTab';
 import ServiceOverlay from '../components/ServiceOverlay';
 import ServicesPage from '../pages/ServicesPage';
+import ServicesTab from '../pages/services/ServicesTab';
 
-let servicesRoutes = {
+let services = {
   type: Route,
   name: 'services',
-  path: 'services/?',
   handler: ServicesPage,
   children: [
     {
       type: Route,
-      name: 'services-detail',
-      path: ':id/?'
+      name: 'services-page',
+      path: '/services/?',
+      children: [
+        {
+          type: Route,
+          name: 'services-all',
+          path: 'all/',
```

Comment:
Sure, we can do that. The reason we didn't already was that the route to the deployments tab is `/services/deployments`, do there's a potential conflict if a user adds a service named `deployments`. What's your preferred deployments route?